ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -78,6 +79,26 @@
   // the count of renegotiation after initial handshake done
   public final Counter sslRenegotiationCount;
 
```

Comment:
I will take it up once this patch goes in, I don't wanna drag this anymore. already its been 3 weeks for this patch.